From de9864a01cdd6ef9f27305990a34e9ca5b7baa5b Mon Sep 17 00:00:00 2001
From: Honza Pokorny <me@honza.ca>
Date: Thu, 6 Jul 2017 14:40:03 -0300
Subject: [PATCH] [tripleo] Add ui logs (#1052)

We're using the "openstack" command to stream the ui log file to stdout.

Signed-off-by: Honza Pokorny <honza@redhat.com>
Signed-off-by: Adam Stokes <battlemidget@users.noreply.github.com>
---
 sos/plugins/openstack_instack.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sos/plugins/openstack_instack.py b/sos/plugins/openstack_instack.py
index f990476..a2c1617 100644
--- a/sos/plugins/openstack_instack.py
+++ b/sos/plugins/openstack_instack.py
@@ -83,6 +83,9 @@ class OpenStackInstack(Plugin):
                             deployment + ".log",
                             timeout=600)
 
+            self.add_cmd_output("openstack object save "
+                                "tripleo-ui-logs tripleo-ui.logs --file -")
+
     def postproc(self):
         protected_keys = [
             "UNDERCLOUD_TUSKAR_PASSWORD", "UNDERCLOUD_ADMIN_PASSWORD",
-- 
2.7.5

